temba/settings_common.py

```diff
@@ -1025,6 +1025,10 @@ def traces_sampler(sampling_context):  # pragma: no cover
         "task": "generate_missing_gif_thumbnails",
         "schedule": crontab(hour=3, minute=30),
     },
+    "update-es-contacts-index-with-channels-and-flows": {
+        "task": "upload_channels_and_flows_to_es_contacts",
+        "schedule": crontab(hour=1, minute=30),
```

Comment:
1:30 AM UTC is about 6:30 PM PST from the day before, so what do you think about making it at 10 PM PST (5 AM UTC)?

Reply:
It makes sense. I'll change it to 5:00.

Reply:
Updated